topics/metagenomics/tutorials/shotgun-metagenomics-data-tutorial/tutorial.md

```diff
@@ -0,0 +1,381 @@
+1. Trimming reads to remove bases that were sequenced with low certainty (= low-quality bases) at the ends of the reads
+2. Removing reads of overall bad quality.
+3. Removing reads that are too short to be informative in downstream analysis
+
+> ### {% icon question %} Questions
+>
+> What are the possible tools to perform such functions?
+>
+>   > ### {% icon solution %} Solution
+>   >
+>   > There are many tools such as **Cutadapt**, **Trimmomatic**, **Trim Galore**, **Clip**, **trim putative adapter** sequences. etc. We choose here **Cutadapt** because it is error tolerant, it is fast and the version is pretty stable.
+>   {: .solution}
+>
+{: .question}
+
+There are several tools out there that can perform these steps, but in this analysis we use **Cutadapt** ({% cite martin2011cutadapt %}).
+
+**Cutadapt** also helps find and remove adapter sequences, primers, poly-A tails and/or other unwanted sequences from the input FASTQ files. It trims the input reads by finding the adapter or primer sequences in an error-tolerant way. Additional features include modifying and filtering reads.
+
+
+> ### {% icon hands_on %} Hands-on: Read trimming and filtering
+>
+> 1. {% tool [Cutadapt](toolshed.g2.bx.psu.edu/repos/lparsons/cutadapt/cutadapt/3.4+galaxy2) %} with the following parameters to trim low quality sequences:
+>    - *"Single-end or Paired-end reads?"*: `Single-end`
+>       - {% icon param-files %} *"FASTQ/A file"*: imported file: `SRR606451_pampa.fasta`
+>
+>    - In *"Filter Options"*
+>       - *"Minimum length"*: `150`
+>    - In *"Read Modification Options"*
+>       - *"Quality cutoff"*: `20`
+>    - In *"Output selector"*
+>       - *"Report"*: select `Report: Cutadapt's per-adapter statistics.`
+>
+>      {% include topics/sequence-analysis/tutorials/quality-control/trimming_question.md %}
+>
+> 2. **Rename** {% icon galaxy-pencil %}
+>     - `Read 1 output` to `QC controlled reads`
+>
+>    {% snippet faqs/galaxy/datasets_rename.md %}
+>
+{: .hands_on}
+
+**Cutadapt** {% icon tool %} outputs a report file containing some information about the trimming and filtering it performed.
+
+> ### {% icon question %} Questions
+>
+> Inspect the output report from **Cutadapt** {% icon tool %}.
+>
+> 1. What % reads contain adapter?
+> 2. What % reads have been trimmed because of bad quality?
+> 3. What % reads have been removed because they were too short?
+>
+>   > ### {% icon solution %} Solution
+>   >
+>   > 
+>   {: .solution}
+>
+{: .question}
+
+## Ribosomal RNA fragments filtering
+
+The ribosomal RNA **rRNAs** are useful for the taxonomic assignment (i.e. which organisms are found) but they do not provide any functional information, (i.e. which genes are expressed) To make the downstream functional annotation faster, we will sort the rRNA sequences using **SortMeRNA** ({% cite kopylova2012sortmerna %}). It can handle large RNA databases and sort out all fragments matching to the database with high accuracy and specificity:
```

Comment:
```suggestion
Ribosomal RNAs (rRNAs) are useful for the taxonomic assignment (i.e. which organisms are found) but they do not provide any functional information (i.e. which genes are expressed). To make the downstream functional annotation faster, we will sort the rRNA sequences using **SortMeRNA** ({% cite kopylova2012sortmerna %}). It can handle large RNA databases and sort out all fragments matching to the database with high accuracy and specificity:
```